### ISSUE TITLE
Autobuilds should be easy to temporarily disable.

### ISSUE THREAD

**Issue by markan** [habitat-sh/builder#1423]
### Describe the Enhancement:
As the maintainer of an origin, I would like to turn off autobuild globally with a single call, and then revert it without loosing the state of individual package autobuilds

As the approver (maybe submitter) of a PR I would like to turn off autobuilds for that PR. 
Some nuance is required around the security implications of allowing arbitrary users to bypass the CI process. More thought may be needed around the implications of this, and may be better split to a separate issue.

### Describe the Need:

Sometimes as a origin maintainer you want temporarily turn off autobuilds for a package or origin.

For example, you may have made a bulk change to plans (add tests or docs for example) that shouldn't trigger an autobuild, at least not until you want it.

Some particular PRs should merge, but not trigger an autobuild; however other PRs in flight will want an autobuild.

### Current Alternative
Packages can individually be enabled/disabled for autobuild, but that requires munging and restoring the autobuild state each time it is used; this is painful if you wish to temporarily suppress a whole origin.

Similarly, avoiding autobuild for a single PR requires an out of band change to builder that must be reset afterwards.

Aha! Link: https://chef.aha.io/features/APPDL-119
